tests/integration/test_writer.py

```diff
@@ -141,3 +145,59 @@ def test_write_mysql(self):
             '?user=root&password='
         )
         self.assertDataFrameEqual(df, TEST_DATA)
+
+
+class TestWriteKafka(SparklyGlobalContextTest):
```

Comment:
it isn't it the same as a test for reader? I thought the goal of the test it to check that data in Kafka are actually placed as expected. Something like:
- write data with sparkle, read them with python client to check that data are placed and serialised as expected
- write data with python client, read them with sparkle to check that we deserialise them properly

Otherwise we can miss internal self-exclusion errors.